.github/workflows/nightly-build.yaml

```diff
@@ -0,0 +1,72 @@
+name: Nightly Builds # nightly job to run against Quarkus master so the issues are detected early.
+on: 
+  schedule:
+    - cron: '0 0 * * *'  # every day at midnight
+
+jobs:
+  nightly-builds:
+    name: Nightly Builds
+    runs-on: ubuntu-18.04    
+    if: github.repository_owner == 'Apicurio' && github.ref == 'refs/heads/master' 
+    steps:
+            
+        - name: Set up JDK 11
+          uses: AdoptOpenJDK/install-jdk@v1
+          with:
+            version: '11'
+            architecture: x64
+        
+        - name: Build Quarkus
+          run: git clone https://github.com/quarkusio/quarkus.git && cd quarkus && mvn -B clean install -DskipTests -DskipITs            
+        
+        - name: Checkout Code
+          uses: actions/checkout@v2
+            
+        - name: Build Registry
+          run: ./mvnw clean install -Pprod -Psql -Pstreams -Pkafkasql -Pmultitenancy -Pinfinispan -Dquarkus.version=999-SNAPSHOT
+
+        - name: Set test profile to all
+          run: echo "test_profile=all" >> $GITHUB_ENV
+
+        - name: Build integration-tests-common
+          run: ./mvnw install -Pintegration-tests -pl integration-tests/integration-tests-common -Dquarkus.version=999-SNAPSHOT
+
+        - name: Run Integration Tests - streams
+          run: ./mvnw verify -Pintegration-tests -P${{ env.test_profile }} -Pstreams -pl integration-tests/testsuite -Dmaven.javadoc.skip=true --no-transfer-progress -Dquarkus.version=999-SNAPSHOT
+
+        - name: Run Integration Tests - sql
+          run: ./mvnw verify -Pintegration-tests -P${{ env.test_profile }} -Psql -pl integration-tests/testsuite -Dmaven.javadoc.skip=true --no-transfer-progress -Dquarkus.version=999-SNAPSHOT
+
+        - name: Run Integration Tests - infinispan
+          run: ./mvnw verify -Pintegration-tests -P${{ env.test_profile }} -Pinfinispan -pl integration-tests/testsuite -Dmaven.javadoc.skip=true --no-transfer-progress -Dquarkus.version=999-SNAPSHOT
+
+        - name: Run Integration Tests - kafkasql
+          run: ./mvnw verify -Pintegration-tests -P${{ env.test_profile }} -Pkafkasql -pl integration-tests/testsuite -Dmaven.javadoc.skip=true --no-transfer-progress -Dquarkus.version=999-SNAPSHOT
+
+        - name: Run Integration Tests - multitenancy
+          run: ./mvnw verify -Pintegration-tests -Pmultitenancy -Psql -pl integration-tests/testsuite -Dmaven.javadoc.skip=true --no-transfer-progress -DtrimStackTrace=false -Dquarkus.version=999-SNAPSHOT 
```

Comment:
@famartinrh are there separate integration tests for MT?  Or should this iteration be removed?

Reply:
there is :) , for now we have only one test, because multitenancy requires special setup the test is in it's specific profile